packages/components/src/button/button.tsx

```diff
@@ -7,11 +7,11 @@ type ButtonProps = React.DetailedHTMLProps<
   React.ButtonHTMLAttributes<HTMLButtonElement>,
   HTMLButtonElement
 > & {
-  variant: "primary" | "secondary";
+  variant?: "primary" | "secondary";
 };
 
 function Button(props: ButtonProps): JSX.Element {
-  const { className = "", variant, ...rest } = props;
+  const { className = "", variant = "primary", ...rest } = props;
```

Comment:
I know this component is really just here for testing purposes, but figured we may as well be consistent in defining default props.